src/process/basic_process.cpp

```diff
@@ -214,10 +214,9 @@ void mp::BasicProcess::setup_child_process()
 void mp::BasicProcess::handle_started()
 {
     pid = process.processId(); // save this, so we know it even after finished
-    const auto& program = qUtf8Printable(process_spec->program());
-
+    const auto program = process_spec->program().toStdString();
     mpl::log(mpl::Level::debug, program,
-             fmt::format("[{}] started: {} {}", pid, program, qUtf8Printable(process_spec->arguments().join(' '))));
+             fmt::format("[{}] started: {} {}", pid, program, process_spec->arguments().join(' ')));
```

Comment:
Why remove `qUtf8Printable`s?

Reply:
Hey @ricab, so, the symptom was that the `program` variable was not printing out expected values and instead, had *some* data that was in `process_spec->arguments().join(' ')`.  This really had me scratching my head as to how in the world this was occurring.  I did some digging and found that since `program` was set by the `qUtf8Printable(process_spec->program())` call previously, the next `qUtf8Printable(process_spec->arguments().join(' ')` call would modify the `program` pointer.  I'm not really sure if this intended behavior in Qt or a bug, but I found the best way to get around this is just to use `std::string`s instead and let the `CString` class handle that.

Reply:
OK, bit of an enigma I see. Anyway, I think it's good like this.